src/ui/notificationsManager.ts

```diff
@@ -1,11 +1,44 @@
 import * as utils from "../utils";
 
+const NOTIFICATION_SOUND_KEYWORDS = ['!important!'];
+const NOTIFICATION_SOUND_URL = 'https://files.catbox.moe/c9nwlu.mp3';
```

Comment:
i'm not sure how i feel about this being hardcoded. maybe it's fine for now but in the future i would definitely want to make this an option